### ISSUE TITLE
2023.03-rc  Failed to get notification in the mastodon APP.

### ISSUE THREAD

**Issue by Tslmuun** [friendica/friendica#12970]
- [x] I have searched open and closed issues for duplicates

``` 
Uncaught Exception TypeError: "Friendica\Object\Api\Mastodon\Status\FriendicaExtension::__construct(): Argument #1 ($title) must be of type string, null given, called in src/Factory/Api/Mastodon/Status.php on line 306" at src/Object/Api/Mastodon/Status/FriendicaExtension.php line 75

TypeError: Friendica\Object\Api\Mastodon\Status\FriendicaExtension::__construct(): Argument #1 ($title) must be of type string, null given, called in src/Factory/Api/Mastodon/Status.php on line 306 and defined in src/Object/Api/Mastodon/Status/FriendicaExtension.php:75
Stack trace:
#0 src/Factory/Api/Mastodon/Status.php(306): Friendica\Object\Api\Mastodon\Status\FriendicaExtension->__construct()
#1 src/Factory/Api/Mastodon/Status.php(294): Friendica\Factory\Api\Mastodon\Status->createFromUriId()
#2 src/Factory/Api/Mastodon/Notification.php(66): Friendica\Factory\Api\Mastodon\Status->createFromUriId()
#3 src/Module/Api/Mastodon/Notifications.php(150): Friendica\Factory\Api\Mastodon\Notification->createFromNotification()
#4 src/BaseModule.php(238): Friendica\Module\Api\Mastodon\Notifications->rawContent()
#5 src/Module/BaseApi.php(105): Friendica\BaseModule->run()
#6 src/App.php(700): Friendica\Module\BaseApi->run()
#7 index.php(52): Friendica\App->runFrontend()
#8 {main}
```

### Platform Info

Friendica Version: 2023.03-rc

PHP version: 8.0.27

SQL version:1518/1518
![Screenshot_20230402-144228](https://user-images.githubusercontent.com/50679343/229337400-64545bbf-5b43-4a24-8571-fc6f95fe78df.png)

**Comment by MrPetovan** [comment]
Hi and thank you for your report! I'm not sure the fatal error you reported is directly related to the app error you're experiencing, so I went ahead and fixed the error in https://github.com/friendica/friendica/pull/12971, and I will ask you to report on the success of the fix regarding your app error once the node you are interfacing with is updated.

**Comment by Tslmuun** [comment]
> Hi and thank you for your report! I'm not sure the fatal error you reported is directly related to the app error you're experiencing, so I went ahead and fixed the error in #12971, and I will ask you to report on the success of the fix regarding your app error once the node you are interfacing with is updated.


Thank you for your work. I will update the server after #12971 has been merged into 2023.03-rc.

**Comment by Tslmuun** [comment]
> > Hi and thank you for your report! I'm not sure the fatal error you reported is directly related to the app error you're experiencing, so I went ahead and fixed the error in #12971, and I will ask you to report on the success of the fix regarding your app error once the node you are interfacing with is updated.
> 
> Thank you for your work. I will update the server after #12971 has been merged into 2023.03-rc.

After testing, this error has been fixed in #12971.  This issue can be closed.Thank you.

**Comment by nupplaphil** [comment]
Fixed with #12971
